Client.hpp

```diff
@@ -0,0 +1,64 @@
+#pragma once
+#include <winsock2.h>
+#include <ws2tcpip.h>
+#include <iostream>
+#include <string>
+#include <thread> 
+#include "Pdu.hpp"
+#include <cstdint>
+#include <unordered_map>
+#pragma comment(lib,"ws2_32.lib")
+
+
+class Client
+{
+private:
+	WSADATA _ws;
+	SOCKET _client_fd;
+	struct sockaddr_in _clientAddr;
+	std::string _color;
+	std::string _username;
+	const std::string RESET_COLOR = "\033[0m";
```

Comment:
for constatns you'll want to use the auto keyoword
for example
`static constexpr auto COLOR = "red";`
or
`static constexpr auto NUMBER = 5;`

auto tells the compiler to deduce the parameter's type during compilation